packages/lib/date-ranges.ts

```diff
@@ -132,12 +132,9 @@ export function processDateOverride({
   };
 }
 
+// This function processes out-of-office dates and returns a date range for each OOO date.
 function processOOO(outOfOffice: Dayjs, timeZone: string) {
-  const utcOffset = outOfOffice.tz(timeZone).utcOffset();
-  const utcDate = outOfOffice.subtract(utcOffset, "minute");
-
-  const OOOdate = utcDate.tz(timeZone);
-
+  const OOOdate = outOfOffice.tz(timeZone, true);
```

Comment:
This does the same thing as substracting the utcOffset - it sets the timeZone without modifying the underlying date. Simplifies the function (also not directly responsible for the test fix, issue was in the test)